packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-visitor.test.ts

```diff
@@ -862,11 +868,11 @@ describe('AppSyncModelVisitor', () => {
     it('Should correctly convert the model map of a simple manyToMany', () => {
       const visitor = createAndGeneratePipelinedTransformerVisitor(simpleManyToManySchema);
 
-      expect(visitor.models.Human.fields.length).toEqual(5);
```

Comment:
Why is this test changing? It looks like you're default to the FF as false, so I'd expect this wouldn't be updated.

Reply:
That change is related to fixing the bug of always generating `id` field when custom PK is defined. In the `Human` type the primary key is `governmentID`. The removed field is `id` which was generated redundantly due to the bug.

Reply:
I don't think this is a breaking change because
1. The GQL transformer does not force to generate `id` field in custom pk schema or the `id` column in DynamoDB
2. The old datastore does not support custom pk schema. Even if the users have custom pk schema defined, they will get into trouble by using datastore which force them to change the schema to non-custom ones

Reply:
Just realize that for users who take modelgen files for non-datastore usage will see this as a breaking change. I make this change behind the feature flag.